src/main/java/qna/domain/Question.java

```diff
@@ -80,12 +87,17 @@ public Question setDeleted(boolean deleted) {
         return this;
     }
 
-    public boolean isDeleted() {
-        return deleted;
+    public List<DeleteHistory> delete() {
+        this.deleted = true;
+        List<DeleteHistory> deleteHistories = new ArrayList<>();
+        deleteHistories.add(new DeleteHistory(ContentType.QUESTION, this.getId(), this.getWriter(), LocalDateTime.now()));
+        List<DeleteHistory> answersDeleteHistories = answers.deleteAll();
+        deleteHistories.addAll(answersDeleteHistories);
+        return deleteHistories;
```

Comment:
줄 바꿈을 적절히 추가하면  조금 더 가독성이 좋아질 것 같네요

```suggestion
        List<DeleteHistory> deleteHistories = new ArrayList<>();
        deleteHistories.add(new DeleteHistory(ContentType.QUESTION, this.getId(), this.getWriter(), LocalDateTime.now()));
        
        List<DeleteHistory> answersDeleteHistories = answers.deleteAll();
        deleteHistories.addAll(answersDeleteHistories);
        
        return deleteHistories;
```

메소드 추출과 Stream을 활용해 코드를 줄일  순 있지만 사실 가독성 측면에서는 지금이 더 나을 것 같아 별도로 코멘트 남기지  않겠습니다~